src/sqlancer/questdb/QuestDBBugs.java

```diff
@@ -3,7 +3,8 @@
 public final class QuestDBBugs {
 
     // https://github.com/questdb/questdb/issues/2689
-    public static boolean bug2689 = true;
+    // set this to false since this error breaks db metadata and interrupts all succeeding query exeuctions
+    public static boolean bug2689;
```

Comment:
https://github.com/questdb/questdb/issues/2689 It seems that the bug was fixed, so we can remove this flag?